ONSdigital/eq-questionnaire-runner
# Add more context to list item tick icons

### What is the context of this PR?
Tick icons on lists of items with completed questions for each item (which are not hubs) do not have enough context for screen readers. So it was decided to add an element (`iconVisuallyHiddenText`) that tells the user that an item is complete, i.e. you have answered all the necessary questions.

The screen reader now says "Section Complete" when it sees a tick icon in the correct context.

This PR also included an update to the DS from 70.0.2 --> 70.0.5

### How to review
- Ensure the changes made are suitable
- Launch `test_list_collector_content_page`, progress to where you have added a couple of companies and you are on the summary of those companies, complete the questions on one of the companies. When you have done that one of the companies should now have a tick against it. Turn VoiceOver on (Shortcut on Mac is `CMD+fn+f5`). Once VoiceOver is on ensure you refresh the page in order for the screen reader to read the correct elements. As the VoiceOver is going over the page, once it reads the tick icon, it should say "Completed".

### Checklist
